apps/api-journeys/prisma/migrations/20211014003622_add_trigger_block_to_block_type/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterEnum
+ALTER TYPE "BlockType" ADD VALUE 'TriggerBlock';
```

Comment:
```suggestion
ALTER TYPE "BlockType" ADD VALUE 'VideoTriggerBlock';
```

since this block is pretty much only going to be used by video.